docs/documentation/getting_started/sharp-bits.md

```diff
@@ -97,3 +97,22 @@ Use `dq.dag(x)` or `x.mT.conj()` instead of `x.dag()` to get the hermitian conju
 ## Using a for loop
 
 If you want to simulate multiple Hamiltonians or initial states, you should use batching instead of a `for` loop. This functionality is explained in detail in the [Batching simulations](../basics/batching-simulations.md) tutorial, together with the associated gain in performance.
+
+## Computing the gradient for complex parameters
+
+To optimize a real-valued function of complex parameters $f:\mathbb{C}^p\to\mathbb{R}$, you should take a step in the direction of the **conjugate** of the gradient given by JAX. For example, if you use the SciPy optimizers with `scipy.minimize()`, you need to conjuguate the gradient before providing it to the optimiser:
```

Comment:
```suggestion
To optimize a real-valued function of complex parameters $f:\mathbb{C}^p\to\mathbb{R}$, you should take a step in the direction of the **conjugate** of the gradient given by JAX. For example, if you use the SciPy or Optax optimizers with `scipy.minimize()` or `optax.lbfgs().update()`, you need to conjuguate the gradient before providing it to the optimiser:
```

Reply:
(not sure about this one, but I like mentionning optax which might be as much used as scipy)

Reply:
Meh, we should put the optax example then. 🤷‍♂️ I think the current example conveys the idea, whatever the optimizer? I chose scipy bc the code is super short.